src/test/scala/tv/teads/wiremock/extension/ExtensionSpec.scala

```diff
@@ -14,4 +16,12 @@ trait ExtensionSpec extends FlatSpec with Matchers with BeforeAndAfterAll with S
     }
   }
 
+  def validate(request: Future[Response], clue: Any, result: String) = {
```

Comment:
Put clue parameter as a varargs to avoid using a tuple when you want to provide multiple data